scripts/ci/functional-test.sh

```diff
@@ -8,6 +8,3 @@ yarn run build
 
 # push the generated screenshots to Argos-CI.
 yarn run test:argos
-
-# check if the next size of the bundle is inside the threshold
-yarn run test:size
```

Comment:
This is already run by the Bundlesize GitHub integration.

Reply:
Nope this is the command that triggers the GitHub integration.

Reply:
Ah, right! Then I'll add the command directly to the [Travis file scripts](https://github.com/algolia/instantsearch.js/blob/chore/scripts-folders/.travis.yml#L19).